core/src/main/java/dk/alexandra/fresco/suite/dummy/arithmetic/DummyArithmeticProtocolSuite.java

```diff
@@ -72,18 +65,21 @@ public int getMaxBitLength() {
 
       @Override
       public void finishedBatch(int gatesEvaluated, DummyArithmeticResourcePool resourcePool,
-          SCENetwork sceNetwork) throws IOException {}
+          SCENetwork sceNetwork) throws IOException {
+      }
 
       @Override
       public void finishedEval(DummyArithmeticResourcePool resourcePool, SCENetwork sceNetwork)
-          throws IOException {}
+          throws IOException {
+      }
     };
   }
 
   @Override
   public DummyArithmeticResourcePool createResourcePool(int myId, int size, Network network,
       Random rand, SecureRandom secRand) {
-    return new DummyArithmeticResourcePoolImpl(myId, size, network, rand, secRand, modulus);
```

Comment:
@pffrandsen : This breaks all tests which uses the dummy arithmetic suite. Reason being a null pointer from basicNumericContext which is set on init (which requires a resourcepool). That is a loop that never ends well :)
We need the createResourcePool to be independent of having init called or not.